### PR TITLE
hw3 initial commit

### PR DESCRIPTION
**Pull request by i-baumann**

### CODE REVIEW

File: hw/hw3-run.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Download jar file to ensure correct driver
+wget --quiet -c https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-8.0.21.tar.gz -O - | tar -zxvf - mysql-connector-java-8.0.21/mysql-connector-java-8.0.21.jar --strip-components=1
+
+# Would not work with -C option, so jankily moving to jar folder in pyspark
+mv mysql-connector-java-8.0.21.jar ../venv/lib/python3.8/site-packages/pyspark/jars/
```
Comment (jcurry9813) [inline hw/hw3-run.sh:7]: It looks like you were able to figure this out without the help of the post the other day, great job here

File: hw/hw3.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    spark = SparkSession.builder.master("local[*]").getOrCreate()
+
+    # Get db tables
+    ##################################
+
+    url = "jdbc:mysql://localhost:3306/baseball"
+
+    # Enter db creds
+    username = input("Enter database username:")
+    dbpass = getpass.getpass("Enter password for database user:")
```
Comment (jcurry9813) [inline hw/hw3.py:22]: This is very clever, makes ease of use excellent

File: hw/hundred_day_avg_transformer.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import pyspark.sql.functions as F
```
Comment (jcurry9813) [inline hw/hundred_day_avg_transformer.py:1]: Code runs smoothly and gets the job done, great work

**Comment by colinanderson10** [comment]
Really great, I especially like the input method for the username and password. Everything runs perfectly.

File: hw/hw3-run.sh (hunk 1 above)
Comment (dafrenchyman) [reply hw/hw3-run.sh:7]: Agreed!

**Review by dafrenchyman — APPROVED** [review]
Please review notes in canvas

**Pull request merged**